securedrop/manage.py

```diff
@@ -295,13 +295,9 @@ def translate(args):
         else:
             log.warning("no translations found (ok for tests, not otherwise)")
 
-    if args.compile:
+    if args.compile and len(os.listdir(args.translations_dir)) > 1:
```

Comment:
This latter check does nothing since there is a default value for this argument. What logic was this trying to capture?

Reply:
it won't try to compile a translation directory that has only one file in it (i.e. the messages.pot file). This is useful now because there indeed are no translations. And it may also be useful later if a test fixture runs it and has no translation.

Reply:
Ah yes. Makes sense. Approved!